INFO:roboLog:Serial interface configured on /dev/ttyACM0@115200 Pyserisal version: 3.5
INFO:roboLog:System started. System time is: 23:45
INFO:roboLog:Current program: {'progID': '001', 'progName': 'Default', 'pumpStartEvery': 1800, 'pumpRunTime': 20, 'pumpON': '08:00', 'pumpOFF': '22:00', 'lightBrightness': 255, 'RGB': '255 150 255', 'lightsON': '06:00', 'lightsOFF': '21:00', 'progDescription': 'Default program, lights and pump are off during quiet hours. Medium watering'}
INFO:roboLog:Background jobs: [<Job (id=test name=broadcastTime)>, <Job (id=checkL name=checkLights)>, <Job (id=checkP name=activatePump)>]
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:16 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:blueprints:Exception on /telemetry [GET]
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/telemetry/telemetry.py", line 24, in telemetry
    return render_template(
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/telemetry/templates/telemetry-index.j2.html", line 1, in top-level template code
    {% extends "base-standard.j2.html" %}
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/templates/base-standard.j2.html", line 13, in top-level template code
    {% include "nav-standard.j2.html" %}
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/templates/nav-standard.j2.html", line 28, in top-level template code
    <a class="dropdown-item" href="{{ url_for('logtail_bp.getlog') }}">tail</a>
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/app.py", line 2216, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/flask/helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/werkzeug/routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'logtail_bp.getlog'. Did you mean 'logtail_bp.getapplog' instead?
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:21 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:26 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:31 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:36 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:41 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:46 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:46:51 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
INFO:roboLog:Serial interface configured on /dev/ttyACM0@115200 Pyserisal version: 3.5
INFO:roboLog:System started. System time is: 23:46
INFO:roboLog:Current program: {'progID': '001', 'progName': 'Default', 'pumpStartEvery': 1800, 'pumpRunTime': 20, 'pumpON': '08:00', 'pumpOFF': '22:00', 'lightBrightness': 255, 'RGB': '255 150 255', 'lightsON': '06:00', 'lightsOFF': '21:00', 'progDescription': 'Default program, lights and pump are off during quiet hours. Medium watering'}
INFO:roboLog:Background jobs: [<Job (id=test name=broadcastTime)>, <Job (id=checkL name=checkLights)>, <Job (id=checkP name=activatePump)>]
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:12 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:17 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:22 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:27 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:32 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:37 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:42 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:47 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:52 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:47:57 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:48:02 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:48:07 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
ERROR:apscheduler.executors.default:Job "checkLights (trigger: interval[0:00:05], next run at: 2021-03-29 23:48:12 CEST)" raised an exception
Traceback (most recent call last):
  File "/home/paolo/NAS/Software/roboGarden/server/.venv/lib/python3.8/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/home/paolo/NAS/Software/roboGarden/server/blueprints/threads.py", line 168, in checkLights
    if int(dataJSON["lightGrowthON"]) == true:
NameError: name 'true' is not defined
